apps/opik-backend/src/main/java/com/comet/opik/domain/SpanDAO.java

```diff
@@ -789,4 +908,20 @@ public Mono<List<WorkspaceAndResourceId>> getSpanWorkspace(@NonNull Set<UUID> sp
                 .collectList();
     }
 
+    public Mono<List<Span>> getByIds(@NonNull List<UUID> ids) {
```

Comment:
This is no longer used, let's remove it. We don't want people to perform this batch retrieval unless is stricly needed. Also, we don't want to leave unused code.

Reply:
Sorry, I forgot about those I will clear the PR